sql/src/test/java/io/crate/execution/dsl/projection/builder/SplitPointsTest.java

```diff
@@ -49,9 +60,11 @@ public void testSplitPointsCreationWithFunctionInAggregation() throws Exception
 
     @Test
     public void testSplitPointsCreationSelectItemAggregationsAreAlwaysAdded() throws Exception {
-        SQLExecutor e = SQLExecutor.builder(clusterService).addTable(T3.T1_DEFINITION).build();
```

Comment:
My preference lately is shifting towards having this per test, to avoid having a `setup` that creates multiple things where each test then only uses a subset.

E.g. now we always process two `addTable`, even though most tests only uses 1.
It's not so bad for unit tests from a speed perspective, but easily adds up in integration tests.

It's fine for me to keep the change, just something to keep in mind.

Reply:
sure, will keep it in mind and avoid described scenarios.